packages/tools/devtools/devtools-browser-extension/e2e-tests/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"compilerOptions": {
+		"target": "es2018",
```

Comment:
We can remove `target` here; the base `ts-common-config.json` already targets `es2020`.